common/minimodel/__init__.py

```diff
@@ -366,15 +367,15 @@ class Atom:
 
 
 class ObservationStatus(Enum):
-    NEW = auto()
-    INCLUDED = auto()
-    PROPOSED = auto()
-    APPROVED = auto()
-    FOR_REVIEW = auto()
-    READY = auto()
-    ONGOING = auto()
-    OBSERVED = auto()
-
+    NEW = 'NEW'
+    INCLUDED = 'INCLUDED'
+    PROPOSED = 'PROPOSED'
+    APPROVED = 'APPROVED'
+    FOR_REVIEW = 'FOR_REVIEW'
+    READY = 'READY'
+    ONGOING = 'ONGOING'
+    OBSERVED = 'OBSERVED'
+    INACTIVE = 'INACTIVE'
```

Comment:
I would prefer not to do this, because the ordering of these makes things much simpler.
We can do lookups by leaving things as is and doing:
```
ObservationStatus['NEW']
```
which returns:
```
<ObservationStatus.NEW: 1>
```
if we stick with `auto()`.

These are considered to have precedence, so we can compare them and in a list of them get the minimum by doing:

```
def min_obs_status(obs_statuses: List[ObservationStatus]): Optional[ObservationStatus] ->
    return min(obs_statuses, default=None)
```